README.md

```diff
@@ -47,163 +61,153 @@ You can also tab complete remote names and branch names e.g.: `git pull or<tab>
+## Installation
 
-Note on performance: Displaying file status in the git prompt for a very large repo can be prohibitively slow.
-Rather than turn off file status entirely (`$GitPromptSettings.EnableFileStatus = $false`), you can disable it on a repo-by-repo basis by adding individual repository paths to `$GitPromptSettings.RepositoriesInWhichToDisableFileStatus`.
+These installation instructions, as well as rest of this readme, applies only to version 1.x of posh-git.
+For v0.x installation instructions see this [README][v0-readme].
 
-## Installation
 ### Prerequisites
+
 Before installing posh-git make sure the following prerequisites have been met.
 
-1. PowerShell 3.0 or higher. Check your PowerShell version by executing `$PSVersionTable.PSVersion`.
+1. Windows PowerShell 5.x or PowerShell Core 6.0.
+   You can get PowerShell Core 6.0 for Windows, Linux or macOS from [here][pscore-install].
+   Check your PowerShell version by executing `$PSVersionTable.PSVersion`.
 
-2. Script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
+2. On Windows, script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
    Check the script execution policy setting by executing `Get-ExecutionPolicy`.
-   If the policy is not set to one of the two required values, run PowerShell as Administrator and execute `Set-ExecutionPolicy RemoteSigned -Scope CurrentUser -Confirm`.
+   If the policy is not set to one of the two required values, run PowerShell as Administrator and
+   execute `Set-ExecutionPolicy RemoteSigned -Scope CurrentUser -Confirm`.
 
 3. Git must be installed and available via the PATH environment variable.
    Check that `git` is accessible from PowerShell by executing `git --version` from PowerShell.
-   If `git` is not recognized as the name of a command verify that you have Git installed.
+   If `git` is not recognized as the name of a command, verify that you have Git installed.
    If not, install Git from [https://git-scm.com](https://git-scm.com).
-   If you have Git installed, make sure the path to git.exe is in your PATH environment variable.
+   If you have Git installed, make sure the path to git is in your PATH environment variable.
 
-### Installing posh-git via PowerShellGet
-If you are on PowerShell version 5 or higher, execute the command below to install from the [PowerShell Gallery](https://www.powershellgallery.com/):
+### Installing posh-git
 
-```powershell
-PowerShellGet\Install-Module posh-git -Scope CurrentUser
-```
-You may be asked if you trust packages coming from the PowerShell Gallery. Answer yes to allow installation of this module to proceed.
+posh-git is available on the [PowerShell Gallery][psgallery-beta1] and can be installed using the PowerShellGet module.
```

Comment:
Thought I had commented on this before, but I guess not…

Let's name this link `psgallery-v1` so we just need to update link once 1.0 final drops.